features/step_definitions/GrantToken_WhenSteps.cpp

```diff
@@ -96,7 +100,11 @@ WHEN("^I attempt to grant a token specifying those permissions$")
 
     pubnub::context pn(context->pubKey, context->subKey);
 
+  // #ifdef MOCK_SERVER_DOCKER
```

Comment:
What's your plan for this commented out lines?

Reply:
Lines are gone now.